### ISSUE TITLE
es_manage usage

### ISSUE THREAD

**Issue by dawnfuel** [jaddison/django-simple-elasticsearch#8]
usage examples/guidelines for the es_manage command are needed in the docs

**Comment by jaddison** [comment]
Agreed! One of many documentation updates I need to get going on. Thanks
for the prod.
